src/commons/Skeletons/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import Skeleton from './Skeleton'
+
+function Skeletons({count}) {
+  return <>{Array.from(Array(count || 1).keys()).map((n) =><Skeleton  key={n} />)}</>
```

Comment:
You don't need to create this Fragment you can also do this:
```suggestion
function Skeletons({ count }) {
  return Array.from(Array(count || 1).keys()).map((n) => (
        <Skeleton key={n} />
      ))}
```